azure-pipelines.yml

```diff
@@ -7,6 +7,9 @@ jobs:
 
   strategy:
     matrix:
+      mac-11.0:
+        imageName: 'macOS-11.0'
+        MP_XCODE_APP: '/Applications/Xcode_12.2.app'
```

Comment:
This is based on the info currently at https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11.0-Readme.md; maybe in the near future a symlink for the latest Xcode 12 will be available and/or the default Xcode version on macOS 11.0 will no longer be Xcode 11.x.